samza-core/src/main/scala/org/apache/samza/checkpoint/OffsetManager.scala

```diff
@@ -193,6 +194,7 @@ class OffsetManager(
     stripResetStreams
     loadStartingOffsets
     loadStartpoints
+    resolveStartpointsToStartingOffsets
```

Comment:
Would make sense to call resolveStartpointsToStartingOffsets  from loadStartpoints?

Reply:
Initially had it after loading defaults. Makes better sense to put it in loadStartpoints. Good catch, thanks!